GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt

```diff
@@ -56,6 +59,9 @@
 # Python
 !/**/*.py
 !/**/*.ipynb
+!requirements.txt
+!Pipenv
+!Pipfile.lock
```

Comment:
@rcarroll901 just want to confirm with you that both of these should be un-ignored

Reply:
Resolving as this line is now outdated